Library/Formula/gdm.rb

```diff
@@ -0,0 +1,39 @@
+require "language/go"
+
+class Gdm < Formula
+  desc "Go Dependency Manager (gdm)"
+  homepage "https://github.com/sparrc/gdm"
+  url "https://github.com/sparrc/gdm/archive/v1.0.tar.gz"
+  sha256 "e0dbec8deed27af3ff4cb16782bc38a4fd8eeb0db8e024b3083a00d36f877cab"
+
+  depends_on "go" => :build
+
+  go_resource "golang.org/x/tools" do
+    url "https://go.googlesource.com/tools.git",
+    :revision => "b48dc8da98ae78c3d11f220e7d327304c84e623a"
+  end
+
+  def install
+    ENV["GOPATH"] = buildpath
+    gdm_path = buildpath/"src/github.com/sparrc/gdm"
+    gdm_path.install Dir["*"]
+
+    Language::Go.stage_deps resources, buildpath/"src"
+
+    cd gdm_path do
+      system "go", "build", "-o", "gdm",
+             "-ldflags", "-X main.Version=#{version}"
+    end
+
+    bin.install gdm_path/"gdm"
```

Comment:
You might want to move this into the `cd gdm_path` block and use `bin.install "gdm"`.